Improve generated documentation

tsg opened the pull request:
The Filebeat module all docs follow the same template. This updates
the generated docs to contain this template and mark the section that
need work as TODOs.

tsg commented:
@dedemorton You review on this one would be appreciated.

dedemorton commented on filebeat/scripts/module/docs.asciidoc:
```diff
@@ -1,3 +1,30 @@
+include::../include/configuring-intro.asciidoc[]
+
+TODO: provide an example configuration
+
```
There's an example in config-option-intro.asciidoc that uses the `fileset_ex` attribute to resolve a fileset name:

````
TIP: When you specify a setting at the command line, remember to prefix the
setting with the module name, for example, +{modulename}.{fileset_ex}.var.paths+
instead of +{fileset_ex}.var.paths+.
````

So...I would add a TODO item before line 54.  Something like:

```
TODO: Set the fileset_ex attribute to the name of any fileset in your module. For example:
:fileset_ex: access
```

dedemorton commented on filebeat/scripts/module/docs.asciidoc:
```diff
@@ -1,3 +1,30 @@
+
+include::../include/config-option-intro.asciidoc[]
+
+TODO: document: fileset variables
```
Should we expand the instructions here and provide an example? Or will that confuse users? If you think this would be confusing to module developers, we can provide feedback when the module is submitted for review.

````
TODO: document: fileset variables. If you're describing a variable that's common to other modules, you
can reuse shared descriptions by including the relevant file. For example:
[float]
==== `access` log fileset settings

include::../include/var-paths.asciidoc[]

tsg commented:
@dedemorton While addressing your comments I realised that I can also make use of the `{fileset}` variable to makes things simpler. Can you have another look?

ruflin commented:
@kvch This might affect your PR in https://github.com/elastic/beats/pull/5544
